app/assets/javascripts/application.js

```diff
@@ -12,5 +12,5 @@
 //
 //= require rails-ujs
 //= require activestorage
-//= require turbolinks
+= require turbolinks
```

Comment:
Okay, I'll put them back in and see what happens. I remember it not working (which is why I removed them) but I'll try again (and post up the error I get, if I do - it could have been something else).